packages/x-data-grid/src/models/gridSlotsComponent.ts

```diff
@@ -154,8 +154,7 @@ export interface GridSlotsComponent extends GridBaseSlots, GridIconSlotsComponen
    */
   footerRowCount: React.JSXElementConstructor<GridSlotProps['footerRowCount']>;
   /**
-   * Toolbar component rendered inside the Header component.
-   * @default GridToolbar
```

Comment:
Don't think it makes sense to list a default component if it's only available internally